netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -258,6 +261,51 @@ _ebpf_sock_addr_get_current_logon_id(_In_ const bpf_sock_addr_t* ctx)
     return logon_id;
 }
 
+static int
+_ebpf_sock_addr_set_redirect_context(_In_ const bpf_sock_addr_t* ctx, _In_ void* data, _In_ uint32_t data_size)
+{
+    int return_value = 0;
```

Comment:
Instead of 0, add EBPF_SUCCESS.

Reply:
Following existing code here for helper functions, which use 0 and -1 for success and failure respectively.